app/assets/javascripts/game.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function(){
+  $('.join').on('click', function(){
+    var gameId = //write jquery to get id - hidden field?
+    var currentUserId = //how to get this?
+
+    $.ajax({
+      method: 'POST',
+      url: 'games/' + gameId + '/players/new',
+      dataType: json,
+      data: { 'game_id' : gameId }
+      success: function(newPlayerCount){
```

Comment:
Expected '}' to match '{' from line 6 and instead saw 'success'.<br>Expected ')' and instead saw ':'.<br>Missing semicolon.<br>Missing name in function declaration.<br>'function' is defined but never used.<br>'newPlayerCount' is defined but never used.